extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/repository-selection.test.ts

```diff
@@ -33,6 +33,19 @@ describe("repository selection", () => {
       );
     });
 
+    it("should return throw error when empt remote user defined list is selected", async () => {
+      const dbManager = setUpDbManager({
+        kind: DbItemKind.RemoteUserDefinedList,
+        repos: [],
+      } as any as DbItem);
```

Comment:
How come you need to do `as any` here? Can we avoid it?

Reply:
No clue to be honest, I copied those lines from somewhere else and only removed entries in `repos: []` when it started complaining about not enough overlap.. 🤷‍♀️

Reply:
Can you try removing it to see what happens? We should generally avoid doing `as any` if possible (can discuss on DM why if needed).

Reply:
Found a simple solution after a while!